org.gnome.seahorse.Application.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.gnome.seahorse.Application",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "seahorse",
+    "copy-icon": true,
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        "--device=dri",
+        /* Access for SSH keys */
+        "--filesystem=~/.ssh:create",
+        /* Access for GPG keys */
+        "--filesystem=~/.gnupg:create",
+        /* Secret Service API */
+        "--talk-name=org.freedesktop.secrets",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Needed for importing,exporting from,to keyservers */
+        "--share=network"
+    ],
+    "build-options" : {
```

Comment:
Those build options are now set by default on Fdo 18.08. You can remove them

Reply:
Ah, didn't know that. Fixed here and will do it upstream also. Thanks :-)